src/main/groovy/myFirstApp/TaskController.groovy

```diff
@@ -0,0 +1,49 @@
+package myFirstApp
+
+import com.todo.service.ConnectionI
+import com.todo.service.TaskServicesI
+import org.restlet.resource.Delete
+import org.restlet.resource.Get
+import org.restlet.resource.Post
+import org.restlet.resource.Put
+import org.restlet.resource.ServerResource
+import com.todo.db.sql.tables.pojos.*
+
+import com.google.inject.Inject
+
+class TaskController extends ServerResource{
+
+    private final ConnectionI connService
+    private final TaskServicesI taskServices
+
+    @Inject
+    TaskController(ConnectionI connService, TaskServicesI taskServices){
+        this.connService=connService
```

Comment:
apply formatting.in the java world there is a convention of spacing between two literals